src/pages/home/uploadSong/AddOwnerModal.tsx

```diff
@@ -1,17 +1,49 @@
+import * as Yup from "yup";
```

Comment:
Woot reuse! Should this be pulled out to a common folder, not sure where modals fit in, components or into a common folder under pages? 
I know there was a large discussion on slack about design flow, so I won't belabor that too much here. However. this would then include all the minting tsx related files under the `pages>uploadSong` file path?

The other piece connected to the design flow that I don't think was mentioned in the discussion on slack was the fact that on `UPLOAD A SONG` when the `MINT SONG` selection is selected, the user needs to connect their wallet. This is in the `MintSongModal.tsx` file, is this going to be added in as well if the `Mint.tsx` is kept separate for edit song?

Reply:
I thought about moving it to the components, the reason why i didnt was because i felt more confident that it would change vs not in the near future. I can add it to the components if you all feel that this will not change soon.

Reply:
It should be a pretty easy change to do, for me it was a matter of guessing what was more likely to happen.

Reply:
oh wait, i had not seen your other PR, i will not be making that change as part of this PR as i can see how merge conflicts will arise